scripts/makeNetwork.py

```diff
@@ -251,6 +252,21 @@ def qemuCmd(iid, network, arch, endianness):
     else:
         raise Exception("Unsupported architecture")
 
+    try:
+        conn = psycopg2.connect(database='firmware', user='firmadyne',
+                password='firmadyne', host='127.0.0.1')
+        cur = conn.cursor()
+        cur.execute("UPDATE image SET guest_ip='%s' WHERE id=%d"%(ip,iid))
+        cur.execute("UPDATE image SET netdev='%s' WHERE id=%d"%(netdev,iid))
+        cur.execute("UPDATE image SET network_inferred=true WHERE id=%d"%(iid))
```

Comment:
Must be `%s`, see http://initd.org/psycopg/docs/usage.html#passing-parameters-to-sql-queries